pg_replicate/src/clients/postgres.rs

```diff
@@ -90,6 +77,7 @@ impl ReplicationClient {
             if let Err(e) = connection.await {
                 warn!("connection error: {}", e);
```

Comment:
Add an early return here to log `connection error: ...` as well as `connection terminated successfully`.